### PR TITLE
data: AnkitAnandMastery17

### PR DESCRIPTION
**Pull request by AnkitAnandMastery17**

### CODE REVIEW

File: data/AnkitAnandMastery17.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Ankit Anand",
+  "type": "personal",
+  "bio": "",
+   "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/An46959780Anand",
+      "icon": "FaTwitter"
+    }
```
Comment (swagataroy30) [inline data/AnkitAnandMastery17.json:10]:
```suggestion
    },
```

File: data/AnkitAnandMastery17.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      {
+      "name": "Connect me on LinkedIn",
+      "url": "https://www.linkedin.com/in/ankit-anand-mastery/",
+      "icon": "FaLinkedIn"
+    }
```
Comment (swagataroy30) [inline data/AnkitAnandMastery17.json:15]:
```suggestion
    },
```

**Review by swagataroy30 — COMMENTED** [review]
You need to add a comma at line 10 and 15. Thank You!

File: data/AnkitAnandMastery17.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Ankit Anand",
+  "type": "personal",
+  "bio": "",
```
Comment (swagataroy30) [inline data/AnkitAnandMastery17.json:4]: Your `bio` is empty. You might want to add something to your bio. Thank You!

File: data/AnkitAnandMastery17.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+   "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/An46959780Anand",
+      "icon": "FaTwitter"
+    }
+      {
+      "name": "Connect me on LinkedIn",
+      "url": "https://www.linkedin.com/in/ankit-anand-mastery/",
+      "icon": "FaLinkedIn"
```
Comment (swagataroy30) [inline data/AnkitAnandMastery17.json:14]:
```suggestion
      "icon": "FaLinkedin"
```

**Review by swagataroy30 — COMMENTED** [review]
Fix the typo. It should be `FaLinkedin`. Thank You!

**Review by himanshu1221 — APPROVED** [review]
LGTM :+1:

**Review by krshkun — APPROVED** [review]
LGMT

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thank you for your Contribution! 🎉 

<p align="center"></p>

**Pull request merged**